docs/api-reference/map.md

```diff
@@ -496,156 +501,20 @@ function App() {
 }
 ```
 
-
 #### `mapboxAccessToken`: string
 
 Token used to access the Mapbox data service. See [about map tokens](/docs/get-started/mapbox-tokens.md).
 
-#### `antialias`: boolean
-
-Default: `false`
-
-If `true` , the gl context will be created with [MSAA antialiasing](https://en.wikipedia.org/wiki/Multisample_anti-aliasing), which can be useful for antialiasing custom layers.
-This is `false` by default as a performance optimization.

```

Comment:
Are we relying on the basemap docs for these now? I can see how that reduces maintenance and confusion as the APIs diverge. I don't like how there are multiple places I'll need to look now, but I can't think of a better alternative.